guidelines-expectations.md

```diff
@@ -0,0 +1,86 @@
+# Guidelines & Expectations for ANGUS 2019 Events
```

Comment:
General comments! The content is lovely, I only added a few comments regarding expectations that weren't there already. Nitty gritty stuff: the syntax for the imperative clauses seems to be using "you" as a subject and but the actual subjects of the other sentences are "they" (e.g. "Avoid disrupting behaviors" and "they do XYZ"). I'd keep it consistent and tend to prefer "you" because it feels more personal. Also, some of the bullets end in a period and some don't - I advocate for consistency.

Reply:
THank you!